apps/linear_algebra/benchmarks/CMakeLists.txt

```diff
@@ -35,18 +35,16 @@ target_link_libraries(halide_benchmarks PRIVATE halide_blas)
 list(APPEND BLAS_NAMES halide)
 
 find_package(Eigen3 QUIET)
-if (NOT Eigen3_FOUND)
+if (NOT EIGEN3_FOUND)
```

Comment:
This should really be `if (NOT TARGET Eigen3::Eigen)`

Reply:
OK, I'll switch to using the TARGET instead of the BOOL variable they defined.